apis/artsy.py

```diff
@@ -1,8 +1,11 @@
 import requests
 from typing import Dict
-from constants import artsy_client_id, artsy_client_skey, artsy_api_url
 import logging
+import os
 
+artsy_client_id = os.getenv("ARTSY_CLIENT_ID")
+artsy_client_skey = os.getenv("ARTSY_CLIENT_SKEY")
+artsy_api_url = os.getenv("https://api.artsy.net/api")

```

Comment:
shouldn't this just be the url?
```suggestions
artsy_api_url = "https://api.artsy.net/api"
```